secator/tasks/grype.py

```
@@ -1,24 +1,51 @@
+
 from secator.config import CONFIG
 from secator.decorators import task
 from secator.definitions import (DELAY, FOLLOW_REDIRECT, HEADER,
 							   OPT_NOT_SUPPORTED, PROXY, RATE_LIMIT, RETRIES,
-							   THREADS, TIMEOUT, USER_AGENT, PATH, STRING, URL)
+							   THREADS, TIMEOUT, USER_AGENT, PATH, STRING)
 from secator.output_types import Vulnerability
 from secator.tasks._categories import VulnCode
 
+GRYPE_MODES = [
+    'git',
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)